src/shared/scrapers/PA/index.js

```diff
@@ -0,0 +1,299 @@
+// import datetime from '../../lib/datetime/index.js';
+import * as fetch from '../../lib/fetch/index.js';
+import maintainers from '../../lib/maintainers.js';
+// import * as parse from '../../lib/parse.js';
+import log from '../../lib/log.js';
+import provinceToIso2 from './province-to-iso2.json';
+
+function titleCase(string) {
+  let result = '';
+  for (let i = 0; i < string.length; i++) {
+    if (i === 0 || string.charAt(i - 1) === ' ') {
+      result += string.charAt(i).toUpperCase();
+    } else {
+      result += string.charAt(i).toLowerCase();
+    }
+  }
+  result = result.replace('Del', 'del'); // ugh.
+  return result;
+}
```

Comment:
```suggestion
/**
 * @param {string} inputString
 * @returns {string}
 */
const titleCase = inputString =>
  inputString
    .split(' ')
    .map(([firstLetter, ...otherLetters]) => firstLetter.toUpperCase() + otherLetters.join('').toLowerCase())
    .join(' ')
    .replace('Del', 'del');
```

Also, is it worth moving the `replace Del del` to the calller instead of inside this titlecase function to keep it generic?

Reply:
Haha wow, what a change! I'm fine with this if you promise you ran it and it works. Regarding "del" - Title capitalization is predictably complicated, but more than that, who knows what capitalization anything is using anyway. Really what I'm doing here is "take all caps provinces and convert them to what my ISO2 list has"; so I'm fine moving the "del" conversion to `getProvinceIso2`.